specification/ai/Azure.AI.Projects/agents/tools/tool_resources.tsp

```diff
@@ -92,13 +92,37 @@ model IndexResource {
    * An index connection id in an IndexResource attached to this agent.
    */
   @encodedName("application/json", "index_connection_id")
-  indexConnectionId: string;
+  indexConnectionId?: string;
```

Comment:
This field is optional now (isn't currently in existing SDK)?

Reply:
If this changed in the newest versions, we should probably tag with @madeOptional(Versions.V2025_05_01)